R/assertthat_checks.R

```diff
@@ -0,0 +1,343 @@
+#' Is Date/Date-time?
+#'
+#' Checks if a date or date-time vector was specified
+#'
+#' @param arg The argument to check
+#'
+#' @author Stefan Bundfuss
+#'
+#' @return `TRUE` if the argument is a date or date-time, `FALSE` otherwise
+#'
+#' @keywords check
+#'
+#' @noRd
+#'
+#' @examples
+#' refdate <- lubridate::ymd("2020-01-02")
+#' date <- lubridate::ymd("2020-02-03")
+#' assertthat::assert_that(admiral:::is_date(refdate), admiral:::is_date(date))
```

Comment:
Is this `:::` since this is exported?

Reply:
For all these `is_*()` function I added the `@noRd` tag which means no actual documentation pages will be created. I want to get rid of these functions any, we should no longer make use of `assertthat::assert_that()`.